<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="84746857-a222-4542-bfe8-bd43f17cd685" name="Tordek@TORDEK-PC 2011-01-07 21:37:23" runUser="Tordek-PC\Tordek" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="7cfd8a5e-6f90-483e-897f-91d4db45673e">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Tordek_TORDEK-PC 2011-01-07 21_37_23">
      <DeploymentItem filename="c:\users\tordek\documents\visual studio 2010\Projects\Folha_de_Ponto\Folha_de_Ponto.Dominio\bin\Debug\Folha_de_Ponto.Dominio.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2011-01-07T21:37:23.0818907-02:00" queuing="2011-01-07T21:37:24.1739532-02:00" start="2011-01-07T21:37:24.2659584-02:00" finish="2011-01-07T21:37:25.5500319-02:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="nome_do_cliente_não_pode_ser_vazio" storage="c:\users\tordek\documents\visual studio 2010\projects\folha_de_ponto\folha_de_ponto.teste\bin\debug\folha_de_ponto.teste.dll" id="272aedbe-ec3b-2d32-bbf8-2956aff68731">
      <Execution id="16729962-427d-4b1d-a4f6-06e15f8533db" />
      <TestMethod codeBase="c:/users/tordek/documents/visual studio 2010/Projects/Folha_de_Ponto/Folha_de_Ponto.Teste/bin/Debug/Folha_de_Ponto.Teste.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="Folha_de_Ponto.Teste.Dado_Um_Cliente, Folha_de_Ponto.Teste, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="nome_do_cliente_não_pode_ser_vazio" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="272aedbe-ec3b-2d32-bbf8-2956aff68731" executionId="16729962-427d-4b1d-a4f6-06e15f8533db" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="16729962-427d-4b1d-a4f6-06e15f8533db" testId="272aedbe-ec3b-2d32-bbf8-2956aff68731" testName="nome_do_cliente_não_pode_ser_vazio" computerName="TORDEK-PC" duration="00:00:00.0649828" startTime="2011-01-07T21:37:24.3089609-02:00" endTime="2011-01-07T21:37:25.4450259-02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="16729962-427d-4b1d-a4f6-06e15f8533db">
      <Output>
        <ErrorInfo>
          <Message>Test method Folha_de_Ponto.Teste.Dado_Um_Cliente.nome_do_cliente_não_pode_ser_vazio did not throw expected exception System.Exception.</Message>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>